packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -157,8 +157,8 @@ double ExtractDoubleFromMap(const flutter::EncodableValue& arguments,
 
 WebView::WebView(flutter::PluginRegistrar* registrar, int viewId,
                  flutter::TextureRegistrar* texture_registrar, double width,
-                 double height, flutter::EncodableMap& params)
-    : PlatformView(registrar, viewId),
+                 double height, flutter::EncodableMap& params, void* winHandle)
```

Comment:
```suggestion
                 double height, flutter::EncodableMap& params, void* win_handle)
```

Reply:
Changed same as the name of the platform view.